packages/database/data-migrations/20240320090315_add_form_styling/data-migration-fix.ts

```diff
@@ -0,0 +1,47 @@
+import { Prisma, PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+async function main() {
+  await prisma.$transaction(async (tx) => {
+    const surveysWithProductOverwrites = await tx.survey.findMany({
+      where: {
+        productOverwrites: {
+          not: Prisma.JsonNull,
+        },
+      },
+    });
+
+    for (const survey of surveysWithProductOverwrites) {
+      if (survey.productOverwrites) {
+        const { brandColor, highlightBorderColor, ...rest } = survey.productOverwrites;
```

Comment:
would it make sense to add a check here if `brandColor` & `highlightColor` is defined and only then update the survey. Because otherwise if the data-migration runs multiple times the surveys will be updated again.